LICENSE

```diff
@@ -0,0 +1,166 @@
+
+                   GNU LESSER GENERAL PUBLIC LICENSE

```

Comment:
We use [Apache license in panther analysis](https://github.com/panther-labs/panther-analysis/blob/main/LICENSE.txt), let me double check on what license we should use.

Reply:
Let's go with Apache 2 for this one as well 😄